src/common/lib/fs.js

```diff
@@ -0,0 +1,5 @@
+import {promisifyAll} from 'bluebird';
+
+const fs = promisifyAll(require('fs'));
+
+export default fs;
```

Comment:
Could it make sense to be used from RN as well? 

Otherwise I'll move it to server.